jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md

```diff
@@ -15,7 +15,7 @@ Rather than explain the task again, I might as well quote the reference post tha
 >
 > With the first field as key, second field as value, the key with the max sum is B, with a total of 13.
 >
-> Fields are delimited by a TAB, and there may be any number of fields on a line. The file name and field numbers of the key and value are passed as command line arguments.
+> Fields are delimited by a *tabulation*, and there may be any number of fields on a line. The file name and field numbers of the key and value are passed as command line arguments.
```

Comment:
This is a quote from the original D blog post, so it doesn't make sense to change the content.